app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt

```diff
@@ -57,7 +57,7 @@ class ConceptCardFragmentPresenter @Inject constructor(
 
     binding.conceptCardToolbar.setNavigationIcon(R.drawable.ic_close_white_24dp)
     binding.conceptCardToolbar.setNavigationContentDescription(
-      R.string.concept_card_close_icon_description
```

Comment:
Since this string no longer seems needed, we should remove it.

Reply:
Sure